docs/docusaurus.config.js

```diff
@@ -42,6 +42,10 @@ const config = {
           // Remove this to remove the "edit this page" links.
           editUrl: "https://github.com/immich-app/immich/tree/main/docs/",
         },
+        api: {
+          path: "../server/immich-openapi-specs.json",
+          routeBasePath: "/apidoc"
```

Comment:
I had that at first, but changed it to this because of both the semantics of the path (`/api` might conceptually host an actual api, although this being a docs site that is maybe unlikely), and because the generated docs include a "try it out" tool that makes request to the (unfortunately hardcoded) /api path. That said, happy to change this back to /api if people prefer.